lib/tasks/one_off/clear_lower_tier_with_conviction_flags.rake

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+namespace :one_off do
+  desc "Clear LOWER tier registrations with conviction flags"
+  task clear_lower_tier_with_conviction_flags: :environment do
+    registration_ids = scope
+    registration_ids.each do |id|
```

Comment:
Could this be done using `update_all` and setting `conviction_sign_offs` to `[]` ?

Reply:
`update_all` seems to only be useable with the activerecord query methods, which I can't get to work in this specific case